src/qml/editorwidgets/RelationEditorBase.qml

```diff
@@ -0,0 +1,430 @@
+      relationEditorModel.reload();
+    }
+
+    onOpened: {
+      addingIndicator.running = false;
+    }
+  }
+
+  property Menu childMenu: Menu {
+    id: childMenu
+    title: qsTr("Child Menu")
+    z: 10000 // 1000s are embedded feature forms, use a higher value to insure feature form popups always show above embedded feature formes
+
+    property var entryReferencingFeature: undefined
+    property string entryDisplayString: ""
+    property var entryNmReferencedFeature: undefined
+    property string entryNmReferencedFeatureDisplayMessage: ""
+
+    onAboutToShow: {
+      atlasMenuLoader.enabled = true;
+    }
+
+    width: {
+      let result = 50;
+      let padding = 0;
+      for (var i = 0; i < count; ++i) {
+        let item = itemAt(i);
+        result = Math.max(item.contentItem.implicitWidth, result);
+        padding = Math.max(item.leftPadding + item.rightPadding, padding);
+      }
+      return mainWindow.width > 0 ? Math.min(result + padding, mainWindow.width - 20) : result + padding;
+    }
+
+    topMargin: mainWindow.sceneTopMargin
+    bottomMargin: mainWindow.sceneBottomMargin
+
+    MenuItem {
+      id: copyChildFeatureAttributes
+
+      font: Theme.defaultFont
+      width: parent.width
+      height: 48
+      leftPadding: Theme.menuItemLeftPadding
+      icon.source: Theme.getThemeVectorIcon("ic_copy_black_24dp")
+
+      text: qsTr("Copy Feature Attributes")
+      onTriggered: {
+        clipboardManager.copyFeatureToClipboard(childMenu.entryReferencingFeature, true);
+      }
+    }
+
+    MenuItem {
+      id: printAtlasChildFeature
+      enabled: LayerUtils.isAtlasCoverageLayer(relationEditorModel.relation.referencingLayer)
+      visible: enabled
+
+      font: Theme.defaultFont
+      width: parent.width
+      height: visible ? 48 : 0
+      leftPadding: Theme.menuItemLeftPadding
+      icon.source: Theme.getThemeVectorIcon("ic_print_black_24dp")
+
+      text: qsTr("Print Atlas Feature to PDF")
+      onTriggered: {
+        childMenu.close();
+        showAtlasMenu();
+      }
+    }
+
+    MenuItem {
+      id: deleteChildFeature
+      enabled: true
+      visible: isActionEnabled('DeleteChildFeature')
+
+      font: Theme.defaultFont
+      width: parent.width
+      height: visible ? 48 : 0
+      leftPadding: Theme.menuItemLeftPadding
+      icon.source: Theme.getThemeVectorIcon("ic_delete_forever_white_24dp")
+
+      text: qsTr("Delete Feature")
+      onTriggered: {
+        deleteDialog.referencingFeatureId = childMenu.entryReferencingFeature.id;
+        deleteDialog.referencingFeatureDisplayMessage = childMenu.entryDisplayString;
+        deleteDialog.nmReferencedFeatureId = childMenu.entryNmReferencedFeature !== undefined ? childMenu.entryNmReferencedFeature.id : 0;
+        deleteDialog.nmReferencedFeatureDisplayMessage = childMenu.entryNmReferencedFeatureDisplayMessage;
+        deleteDialog.visible = true;
+      }
+    }
+  }
+
+  Loader {
```

Comment:
The use of a loader here is to defer the cost involved in atlas-related item / model generation until we open the menu.

Reply:
Do you want to leave that as a comment in the file, so someone working on this in the future will directly know?

Reply:
@m-kuhn , done.